File: src/test/java/com/example/ptiringo/ApplicationTests.java
package com.example.ptiringo;

import org.junit.ClassRule;
import org.junit.Test;
import org.junit.runner.RunWith;
import org.springframework.boot.test.context.SpringBootTest;
import org.springframework.test.context.junit4.SpringRunner;

import com.palantir.docker.compose.DockerComposeRule;

@RunWith(SpringRunner.class)
@SpringBootTest
public class ApplicationTests {

	@ClassRule
	public static DockerComposeRule docker = DockerComposeRule.builder()
			.file("docker-compose.yml").build();

	@Test
	public void contextLoads() {
		Thread.currentThread().getContextClassLoader()
				.getResourceAsStream("config.properties");
	}

}

File: src/test/kotlin/com/example/ptiringo/ApplicationTests.kt
package com.example.ptiringo

import org.junit.Test
import org.junit.runner.RunWith
import org.springframework.boot.test.context.SpringBootTest
import org.springframework.test.context.junit4.SpringRunner

@RunWith(SpringRunner::class)
@SpringBootTest
class ApplicationTests {

    @Test
    fun contextLoads() {
        Thread.currentThread().contextClassLoader.getResourceAsStream("config.properties")
    }
}
